sdlib/d/gc/tcache.d

```diff
@@ -1,5 +1,6 @@
 module d.gc.tcache;
 
+import d.gc.bin;
```

Comment:
Yet, Bin is not used in there. This is a strong sign that things aren't organized properly.

Reply:
slabAllocGeometry is, not Bin, hence the comment.